docs/Manual/Deployment/Kubernetes/DeploymentResource.md

```diff
@@ -143,6 +143,19 @@ The encryption key cannot be changed after the cluster has been created.
 The secret specified by this setting, must have a data field named 'key' containing
 an encryption key that is exactly 32 bytes long.
 
+### `spec.networkAttachedVolumes: bool`
+
+The default of this option is `false`. If set to `true`, and the
+deployed ArangoDB version is new enough, a `ResignLeaderShip` operation
```

Comment:
Can we state the actual version? Also I would mention `You must only set this option to `true` if
your persistent volumes are "movable" in the sense that they can be
mounted from a different k8s node, like in the case of network attached
volumes. If your persistent volumes are tied to a specific pod, you
should leave this option on `false`.` First, as it at least for me helped me understand the relation to resignLeadership...